app/javascript/packages/verify-flow/forgot-password/forgot-password.tsx

```diff
@@ -0,0 +1,60 @@
+import { PageHeading } from '@18f/identity-components';
+import { t } from '@18f/identity-i18n';
+import { Button } from '@18f/identity-components';
+import { getAssetPath } from '@18f/identity-assets';
+
+interface ForgotPasswordProps {
+  goBack: () => void;
+}
+
+export function ForgotPassword({ goBack }: ForgotPasswordProps) {
+  const stepIndicator = document.getElementsByTagName('lg-step-indicator')[0];
```

Comment:
We should want to avoid querying the DOM directly from a React component, especially for elements outside the component's own. If we need to, it might be reasonable to implement the step indicator inside each individual step. However, it occurs to me: Maybe we want to show the step indicator for this warning screen? I know we haven't 'til now, but it would match similar behavior we have for the document capture warning pages.

It'd be good to get @Kamal-Munshi and/or @anniehirshman-gsa to weigh in.

Reply:
@nprimak @aduth I don't feel strongly about the step indicator being on this screen or not... we _do_ show it when a user is on an error for a particular step, but we _don't_ show it for the Cancel screen -- same basic page template, but it's a separate process that you could arrive at it from any step. This page could be interpreted either way in my view. Is there an approach that makes more sense from a technical perspective?

Reply:
@anniehirshman-gsa Would you have any problems with showing the step indicator on the Cancel screen as well? Depending if / when we implement that into the React flow, it would probably just be simplest to always show the step indicator.